src/Screen/Fields/Select.php

```diff
@@ -146,6 +147,18 @@ public function empty(string $name = '', string $key = ''): self
         });
     }
 
+    /**
+     * @param bool $value
+     *
+     * @return self
+     */
+    public function allowAdd(bool $value = true): self
```

Comment:
I think we don't have to create this method explicitly. It is enough to define it in phpDoc at the top of the class, as we do in other places for the IDE's autocompletion